sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -180,7 +184,7 @@ export class ECertFileHandler extends ESDCFileHandler {
   ): Promise<ECertUploadResult> {
     try {
       const disbursements =
-        await this.eCertGenerationService.prepareDisbursementsForECertGeneration(
+        await this.eCertGenerationService.getReadyToSendDisbursements(
           offeringIntensity,
           entityManager,
         );
```

Comment:
Question outside the PR. If there are no disbursements to process why do we consider it as error?

Reply:
It can be improved but right now it will cancel the transaction in place stated by the sequence control block.
I am adding a comment to explain it.

Reply:
Added a simple comment for now.